system/system_monitor/include/hdd_reader/hdd_reader.hpp

```diff
@@ -38,6 +68,9 @@ struct HDDInfo
   uint8_t temp_;        //!< @brief temperature(DegC)
   // Lowest byte of the raw value contains the exact temperature value (Celsius degrees)
   // in S.M.A.R.T. information.
+  uint64_t power_on_hours_;           //!< @brief power on hours count
+  uint64_t total_data_written_;       //!< @brief total data written
```

Comment:
What are the units? Times? MB?

Reply:
The units are size, but they differ depending on devices.
In ATA : LBA block size
In NVMe : 512byte * 1,000 (and rounded)
https://nvmexpress.org/wp-content/uploads/NVM-Express-Base-Specification-2.0b-2021.12.18-Ratified.pdf
Figure 207.Data Units Written